Fix displaying hidden resources in global search (#8811)
decidim/decidim: CHANGELOG.md

### Added

#### Moderated content can now be removed from search index

> PR [\#8811](https://github.com/decidim/decidim/pull/8811) is addressing an issue when the moderated resources are not removed from search index.

Reviewer: ```suggestion
PR [\#8811](https://github.com/decidim/decidim/pull/8811) is addressing an issue when the moderated resources are not removed from the general search index.
```

> We have introduced a new task  that would remove the moderated content from being displayed in search. Please run the following command if you want to remove old moderated data from your search results

Reviewer: ```suggestion
This will automatically work for new moderated resources. For already existing ones, we have introduced a new task that will remove the moderated content from being displayed in search:
```

This will automatically work for new moderated resources. For already existing ones, we have introduced a new task that will remove the moderated content from being displayed in search:

> ```ruby

Reviewer: ```suggestion

```ruby
```

> rake decidim:moderation:remove_from_search

Reviewer: Sorry that I didn't tell it yesterday, but this is an idea that I've been thinking in the last couple of months and I think it could be the moment to introduce it… 

As this is a task that should only be executed one time, during the upgrade process, how about if we do it with a namespace, so we have the different tasks ordered by release? Something like:
```bash
rake decidim:upgrade:v0.27:remove_moderations_from_search
```

I don't even know if it's valid to have a dot character (".") in a rake task (or if we could have any kind of problem in shells?). If the dot is a problem, then I guess it could only be `rake decidim:upgrade:remove_moderations_from_search `. 

The idea is to have a bit of consistency here, because at the moment it is a bit messy. What do you think @alecslupu?

Author: What we can do is : 

```
rake decidim:upgrade:v0_27:remove_moderations_from_search
```
